docker/conf/docker-sync-win.yml.dist

```diff
@@ -5,25 +5,24 @@ syncs:
         sync_host_port: 10872
         sync_strategy: 'unison'
         src: './'
-        sync_excludes: [
-            '.ci',
-            '.git',
-            '.github',
-            '.idea',
-            '.docker-sync',
-            '.DS_Store',
-            'docker',
-            'nbproject',
-            'project-base/docker',
-            'project-base/docs',
-            'project-base/kubernetes',
-            'project-base/node_modules',
-            'project-base/var/cache',
-            'project-base/var/elasticsearch-data',
-            'project-base/var/postgres-data',
-            'project-base/web',
-            'vendor'
-        ]
+        sync_excludes:
+            - .ci,
```

Comment:
I think this makes the comma a part of the string (`".ci,"` instead of `".ci"`)

Reply:
again you have right, fixed